packages/web/src/pages/upload-page/fields/stream-availability/usdc-purchase-gated/UsdcPurchaseFields.tsx

```diff
@@ -88,6 +89,18 @@ export const UsdcPurchaseFields = (props: TrackAvailabilityFieldsProps) => {
   const [{ value: downloadConditions }] =
     useField<Nullable<AccessConditions>>(DOWNLOAD_CONDITIONS)
 
+  const [, , { setValue: setAlbumTrackPrice }] =
+    useField<Nullable<number>>(ALBUM_TRACK_PRICE)
+
+  useEffect(() => {
```

Comment:
interesting... i don't have the most formik knowledge, but is there really no better way to do this? you can't set it in `helpers` above?

Reply:
Yeah it feels weird but given the current form structure I dont think there's a better approach. 
`initialValues` on the form don't make sense since `stream_conditions` are null to begin with. 
I could add another `Formik` wrapper to set initial values just for the two price inputs but that seems like overkill.
I don't believe `defaultValue` works in Formik since it uses controlled values.
So then the only thing left is just to set the value programmatically whenever inputs first show up; so on initial render makes sense 🤷